index.html

```diff
@@ -0,0 +1,282 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+
+
+<body>
+  <!-- header Section -->
+  <header>
+    <a href="#" class="logo">Devlo<span>pe</span>r..</a>
+    <ul class="navList open">
+      <li><a href="#home" class="active">Home</a></li>
+      <li><a href="#about" >About Me</a></li>
+      <li><a href="#myProject">MyProject</a></li>
+      <li><a href="#portfolio">Portfolio</a></li>
+      <li><a href="#contact">Contact</a></li>
+    </ul>
+    <div class="bx bx-menu" id="menu-icon">
+      <i class="fa-solid fa-bars"></i>
+    </div>
+  </header>
+
+  <!-- home section -->
+  <section id="home" class="home">
+    <div class="home-text">
+      <div class="slide">
+        <span class="one">Hello</span>
+        <span class="two">I'm</span>
+      </div>
+      <h1>Farwa</h1>
+      <h3>Web <span>Developer</span></h3>
+      <p>
+        Web Developer with one year experience that keep customers <br> coming for about services make best effort
+      </p>
+       <div class="button">
+        <a href="#about" class="btn">About Me</a>
+
+       </div>
+    </div>
+  </section>
+
+
+  <!-- about section -->
+  <section id="about" class="about">
+    <div class="about-img">
+      <img src="./img/about.jpg" alt="this a picture">
+    </div>
+    <div class="about-text">
+      <h2>About <span>Me</span></h2>
+      <h4>Web Developer</h4>
+      <p>
+        Skilled and dedicated software engineer with 1 year of experience focused on frontend development in applications and web portals, JS focused on React.
+        Deep knowledge in web application development using technologies such as react as well as its integration with third-party services.
+        Key areas of technical expertise: SEO of web pages, web application optimization, frontend software design, web solutions layout, passion for sharing my knowledge, responsibility, and commitment as primary values, always looking for learning, proactive
+        I have completed two years of University, not only did I maintain excellent grades.
+      </p>
+        <br>
+    </div>
+  </section>
+
+  <!-- myProject sections -->
+  <section class="services" id="myProject">
+    <div class="main-text">
+      <h2><span>My</span>MyProject</h2>
```

Comment:
- Solve the typo issue `My My project` Remove one ` My`

![Screenshot 2024-06-06 210220](https://github.com/FarwaMuhibZada/Personal_Portfolio/assets/96838030/2daae5f6-9817-49b9-b203-29e72db6dd7b)